lib/github_changelog_generator/generator/generator_processor.rb

```diff
@@ -132,8 +133,8 @@ def include_issues_by_labels(issues)
     # @return [Array] issues without labels or empty array if add_issues_wo_labels is false
     def filter_wo_labels(issues)
       if options[:add_issues_wo_labels]
-        issues_wo_labels = issues.select do |issue|
-          !issue["labels"].map { |l| l["name"] }.any?
+        issues_wo_labels = issues.find_all do |issue|
+          !issue["labels"].map { |l| l["name"] }.one?
```

Comment:
This seems like a turning-around of the PreferredMethods list:

https://github.com/bbatsov/rubocop/blob/master/config/default.yml#L670-L683